frontend/js/src/utils/APIService.ts

```diff
@@ -581,15 +581,24 @@ export default class APIService {
 
   getFeedbackForUserForRecordings = async (
```

Comment:
I think this method and `getFeedbackForUserForMBIDs` can now be merged into a single method.

We would need to make the MSIDs array optional to allow for what getFeedbackForUserForMBIDs does (fetching by MBIDs only):
```typescript
getFeedbackForUserForRecordings = async (
    userName: string,
    recording_mbids: string[], // note that I moved the order of this required parameter
    recording_msids?: string[] // while this one is last since it is optional
  ) => {
```

I'm not certain of the server API endpoint will like `recording_msids: undefined` in the case it's not passed to the method, but you can always add a check if it doesn't like it, something along the lines of:
```
if(recording_msids?.length){
  body.recording_msids = recording_msids;
}
```

Reply:
Okay cool I'll go ahead and merge these two and refactor the endpoints and tests as well for these